app/config.py

```diff
@@ -66,10 +66,10 @@
 # This is only relevant if USE_ENCRYPTED_IDS above is True. If not, this config parameter can be ignored.
 # It is recommended, but not strictly required, for you to replace the string below with the output of os.urandom(32),
```

Comment:
os.urandom(32) will not work, since bytes and str will have different len after .encode() and .decode() (and AES requires constant key len dividable by 2^4)